tools/c7n_azure/c7n_azure/resources/mysql.py

```diff
@@ -0,0 +1,40 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('mysql')
+class MySQL(ArmResourceManager):
+    """Azure MySQL Server Resource
+
+    :example:
+
+    Returns all MySQL servers
+
+    .. code-block:: yaml
+
+        policies:
+          - name: basic-logic-app
+            resource: azure.mysql
+            filters:
+              - type: value
+                key: sku.name
+                op: equal
+                value: Basic
+
+    """
+
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Compute']
```

Comment:
We have databases doc group: https://cloudcustodian.io/docs/azure/resources/group-databases.html

Reply:
oh good catch